src/components/filters/MeasureValueFilter/DropdownAfmWrapper.tsx

```diff
@@ -20,6 +20,8 @@ export interface IDropdownProps {
     locale?: string;
     anchorEl?: EventTarget | string;
     separators?: ISeparators;
+    displayTreatNullAsZeroOption?: boolean;
```

Comment:
please provide default values explicitly for this react component. See src/components/filters/AttributeFilter/AttributeFilter.tsx `public static defaultProps: Partial<IAttributeFilterProps> = {`